pkgs/applications/version-management/gitkraken/default.nix

```diff
@@ -0,0 +1,69 @@
+{stdenv, lib, libXcomposite, libgnome_keyring, makeWrapper, udev, curl, fetchurl, libXfixes, atk, gtk2, libXrender, pango, gnome2, cairo, freetype, fontconfig, libX11, libXi, libXext, libXcursor, glib, libXScrnSaver, libxkbfile, libXtst, nss, nspr, cups, alsaLib, expat , gdk_pixbuf, libXdamage, libXrandr, dbus}:
```

Comment:
Please wrap overlong lines using the following style:

```
{ stdenv, .......
, foo, .....
}:
```

EDIT: semicolon -> colon typo